src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/AnswerRatingOperations.java

```diff
@@ -323,22 +325,25 @@ public Rating updateRating(Rating rating) throws IllegalArgumentException, Illeg
     private void addToExperimentCalibration(int workerID, int experimentId) {
         Supplier<Optional<CalibrationAnswer>> doAdd = () -> calibrationOperations
                 .getCalibrationForExperiment(experimentId)
-                .map(answerOption -> workerCalibrationOperations
-                                .insertAnswer(workerID, answerOption.getIdCalibrationAnswerOption())
-                );
+                .map(answerOption -> {
+                    try {
+                        return workerCalibrationOperations
+                                        .insertAnswer(workerID, answerOption.getIdCalibrationAnswerOption());
+                    } catch (IllegalArgumentException ignored) {
+                        //is ok, the worker was already added
+                        return CalibrationAnswer.getDefaultInstance();
```

Comment:
It should probably return the same thing as `workerCalibrationOperatoins.insertAnswer` would return. `CalibrationAnswer.getDefaultInstance` is just an empty object of that time. In that case `null` would just result in a `204 No Content` then.